scripts/asm-installer/install_asm

```diff
@@ -1848,8 +1848,9 @@ bind_user_to_cluster_admin(){
   info "Querying for core/account..."
   local GCLOUD_USER; GCLOUD_USER="$(gcloud config get-value core/account)"
   info "Binding ${GCLOUD_USER} to cluster admin role..."
+  PREFIX="$(echo "${GCLOUD_USER}" | cut -f 1 -d @)"
```

Comment:
Can you add a `local PREFIX; ` here so that the variable is scoped to this function?

Reply:
Sure, I have added the `local` keyword. It should be fine now. Thanks, @zerobfd, for helping me out.